tests/integration/api_v2/http/feeEstimate.test.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const config = require('../../../config');
+const { api } = require('../../../helpers/api');
+
+const {
+	badRequestSchema,
+} = require('../../../schemas/httpGenerics.schema');
+
+const {
+	feeEstimateSchema,
+	goodRequestSchema,
+	metaSchema,
+} = require('../../../schemas/api_v2/fees.schema');
```

Comment:
Is there a reason why `goodRequestSchema` and `metaSchema` are duplicated within `fees.schema.js` and not used from `httpGenerics.schema.js`?

Also, consider renaming the file to `tests/integration/api_v2/http/fees.test.js` to keep it consistent with the endpoint and the corresponding schema file name.

Reply:
Yes, we have different response structure for fees. In fees response, the data is an object, and meta object is different as well.